base/acme/src/main/java/org/dogtagpki/acme/database/PostgreSQLDatabase.java

```diff
@@ -1135,11 +1165,24 @@ public void removeCertificate(String certID) throws Exception {
         String sql = statements.getProperty("removeCertificate");
         logger.info("SQL: " + sql);
 
-        connect();
```

Comment:
Was this removed by mistake?

Reply:
It's intentional. The `removeCertificate()` is now an internal method in `PostgreSQLDatabase` (I probably need to make it private). It's only used by `removeExpiredCertificates()` which already calls `connect()` so we don't need to call it again in `removeCertificate()`.

Reply:
It's private now.